WebApp/src/components/StudyGroup/StudyGroup.scss

```diff
@@ -0,0 +1,17 @@
+.StudyGroup {
```

Comment:
This part is copy-pasted from home page. I did extracted it to separate component. It is not however merged in develop yet...